srcpkgs/vit/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'vit'
+pkgname=vit
+version=2.1.0
+revision=0
+build_style=python3-module
+hostmakedepends="python3-pip task python3-pytz python3-urwid python3-tzlocal"
+depends="task"
+checkdepends=
+short_desc="VIT is a lightweight, fast, curses-based front end to Taskwarrior "
+maintainer="pinoinha <pinoinha@disroot.org"
+license="MIT"
+homepage="https://github.com/vit-project/vit"
+changelog="https://raw.githubusercontent.com/vit-project/vit/2.x/CHANGES.md"
+distfiles="${homepage}/archive/refs/tags/v${version}.tar.gz"
+checksum=56676651164fb71878a0c1e4bb369cfa336d46e6fd0de7e6167b57fca7eaf662
+
+pre_build() {
+	pip3 install tasklib
```

Comment:
Add `python3-tasklib` to `hostmakedepends`.

Reply:
just as a clarification: is this line needed? in the vit PR a fellow contributor said it shouldn't be essential

also the template already installs the files in the system